perfkitbenchmarker/linux_virtual_machine.py

```diff
@@ -540,6 +540,34 @@ def BurnCpu(self, burn_cpu_threads=None, burn_cpu_seconds=None):
         time.sleep(end_time - time.time())
       self.RemoteCommand('pkill -9 sysbench')
 
+  def PrepareBackgroundWorkload(self):
+    """ Install packages needed for the background workload """
+    if self.background_cpu_threads:
+      self.Install('sysbench')
+
+  def StartBackgroundWorkload(self):
+    """Starts the blackground workload.
+
+    Args:
+      self: The target vm.
+      background_cpu_threads: Number of threads active in the background.
```

Comment:
This isn't actually an argument for this function - it's an attribute of the vm instance.